airflow/models.py

```diff
@@ -1022,12 +1022,21 @@ def are_dependents_done(self, session=None):
     @provide_session
     def previous_ti(self, session=None):
         """ The task instance for the task that ran before this task instance """
```

Comment:
Going to have to dig into this. Apparently, moving to a DR based previous_ti causes issues when NOT using a DR (backfill). Finally figure that out this afternoon. Will attempt to fix tonight.